File: src/WellTool.Cron/hutool-core/src/test/java/cn/hutool/core/util/IssueICOJVZTest.java
package cn.hutool.core.util;

import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

public class IssueICOJVZTest {
	@Test
	void toUnderlineTest(){
		String field = "PAGE_NAME";
		field = StrUtil.toUnderlineCase(field).toUpperCase();
		Assertions.assertEquals("PAGE_NAME", field);
	}
}

File: src/WellTool.Core.Tests/IssueICOJVZTest.cs
using Xunit;
using WellTool.Core;
using WellTool.Core.Text;
using WellTool.Core.Util;

namespace WellTool.Core.Tests
{
    /// <summary>
    /// Issue #ICOJVZ 测试
    /// </summary>
    public class IssueICOJVZTest
    {
        [Fact]
        public void ToUnderlineTest()
        {
            string field = "PAGE_NAME";
            field = NamingCase.ToUnderlineCase(field).ToUpperInvariant();
            Assert.Equal("PAGE_NAME", field);
        }
    }
}
